setup/package.dffedora

```diff
@@ -47,7 +44,6 @@ RUN dnf update -d 0 -e 0 -y && \
 	  python3-xlrd \
 		python3-xlsxwriter \
 		libsass \
-		pytz \
```

Comment:
That seems odd. `pytz` is absolutely in active use.

Reply:
Sure, but it's an artifact of python 2 pytz. python3-pytz is already defined line 36

Reply:
Line 42 since I reorganized the file :-)

Reply:
Ah yes, makes more sense when looking at more than just the small hunk.